packages/components/react/src/core/FileTree.tsx

```diff
@@ -162,14 +163,16 @@ interface ButtonProps {
   children: ReactNode;
   className?: string;
   onClick?: () => void;
+  'aria-pressed'?: boolean;
 }
 
-function NodeButton({ depth, iconClasses, onClick, className, children }: ButtonProps) {
+function NodeButton({ depth, iconClasses, onClick, className, 'aria-pressed': ariaPressed, children }: ButtonProps) {
   return (
     <button
       className={`flex items-center gap-2 w-full pr-2 border-2 border-transparent text-faded ${className ?? ''}`}
       style={{ paddingLeft: `${12 + depth * NODE_PADDING_LEFT}px` }}
       onClick={() => onClick?.()}
+      aria-pressed={ariaPressed === true ? 'true' : undefined}
```

Comment:
This isn't 100% correct either as these are not toggle buttons, but it's better than nothing. Proper way would be to throw these buttons away and create a proper tree: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/